css/style.css

```diff
@@ -30,6 +30,7 @@
   padding-top: 24px;
   text-decoration-line: underline;
   color: #fff;
+  font-variant: small-caps;
```

Comment:
It's okay to try to use font-variant ,but let's stick to the original design of topography 👍🏽